cmd/kyverno/main.go

```diff
@@ -120,6 +124,15 @@ func main() {
 	}
 	flag.Parse()
 
+	if logFormat == "json" {
+		zapLog, err := zap.NewProduction()
```

Comment:
Why are we switching to `zap` here ?
Chances are that we want to stick with `klog`.